### PR TITLE
Make FeeReimbursement.event & refractor

### PR DESCRIPTION
**Pull request by tmb**

### CODE REVIEW

File: app/models/fee_reimbursement.rb [hunk 1]
```diff
@@ -40,6 +40,14 @@ def status_color
     'error'
   end
 
+  def event
+    if donation
+      return donation.event
+    else
+      invoice.event
```
Comment (thesephist) [inline app/models/fee_reimbursement.rb]: Tiny nit, but can we be consistent here and stick to either using `return` keyword or implicit return for both cases?

File: app/views/fee_reimbursements/index.html.erb [hunk 2]
```diff
@@ -23,10 +23,10 @@
       <tr class="<%= 'shade-yellow' if reimbursement.unprocessed? %>">
         <td><%= "##{reimbursement.id}" %></td>
         <% if reimbursement.invoice %>
-          <td><%= link_to "#{reimbursement.invoice.event.name}", reimbursement.invoice.event, class: 'underline' %></td>
+          <td><%= link_to "#{reimbursement.event.name}", reimbursement.event, class: 'underline' %></td>
           <td><%= link_to "##{reimbursement.invoice.id}", reimbursement.invoice, class: 'underline' %></td>
         <% else %>
-          <td><%= link_to "#{reimbursement.donation.event.name}", reimbursement.donation.event, class: 'underline' %></td>
+          <td><%= link_to "#{reimbursement.event.name}", reimbursement.event, class: 'underline' %></td>
```
Comment (thesephist) [inline app/views/fee_reimbursements/index.html.erb]: Can we pull this line out of the if-else case now that it's the same across both cases?

**Review by thesephist — CHANGES_REQUESTED** [review]
Almost there ♨️

File: app/models/fee_reimbursement.rb (hunk 1 above)
Comment (tmb) [reply app/models/fee_reimbursement.rb]: yes

File: app/views/fee_reimbursements/index.html.erb (hunk 2 above)
Comment (tmb) [reply app/views/fee_reimbursements/index.html.erb]: yes

**Review by thesephist — APPROVED** [review]
✌️

**Pull request merged**